easybuild/easyconfigs/t/TensorFlow/TensorFlow-2.0.0-fosscuda-2019b-Python-3.7.4.eb

```diff
@@ -0,0 +1,101 @@
+easyblock = 'PythonBundle'
+
+name = 'TensorFlow'
+version = '2.0.0'
+versionsuffix = '-Python-%(pyver)s'
+
+homepage = 'https://www.tensorflow.org/'
+description = "An open-source software library for Machine Intelligence"
+
+toolchain = {'name': 'fosscuda', 'version': '2019b'}
+toolchainopts = {'usempi': True}
+
+builddependencies = [
+    ('Bazel', '0.26.1'),
+    ('protobuf', '3.7.1'),
```

Comment:
@Flamefire I noticed we're using an old `protobuf` as a build dep to build the latest `protobuf` Python bindings (3.10.0, see below in `exts_list`).

That's probably not the best idea, shall we bump this to 3.10.0 as well (which is provided by #9280)?

Reply:
Sounds like a good idea. I did not consider updating this especially as protobuf is not mentioned in the TF dependencies in their manual.

Reply:
OK, I'll look into that.

I'll also switch the `git` build dep to the `-nodocs` variant, since building the `git` docs required stuff in the OS that is not that trivial to provide via EasyBuild.